tla-pp/src/main/scala/at/forsyte/apalache/tla/pp/passes/ConfigurationPassImpl.scala

```diff
@@ -43,40 +45,72 @@ class ConfigurationPassImpl @Inject()(val options: PassOptions,
     * @return true, if the pass was successful
     */
   override def execute(): Boolean = {
-    var module = tlaModule.get
+    // try to read from the TLC configuration file
+    val configuredModule = tryConfigureFromTlcConfig(tlaModule.get)
 
+    // dump the configuration result
+    val outdir = options.getOrError("io", "outdir").asInstanceOf[Path]
+    PrettyWriter.write(configuredModule, new File(outdir.toFile, "out-config.tla"))
+
+    // set fallback options
+    setFallbackOptions()
+
+    // make sure that the operators are defined
+    ensureDeclarationsArePresent(configuredModule)
+
+    outputTlaModule = Some(configuredModule)
+    true
+  }
+
+  // if checker.init and checker.next are not set, set them to Init and Next, respectively
+  private def setFallbackOptions(): Unit = {
+    if (options.get("checker", "init").isEmpty) {
+      logger.info("  > Option --init is not set. Using Init")
```

Comment:
This seems to me like natural and expected behavior, and not very informative. I worry that this adds noise without much benefit. So I'd suggest instead including information about this defaulting behavior into the CLI help, and demoting this logging to

```suggestion
      logger.debug("  > Option --init is not set. Using Init")
```

Reply:
I was under the impression that the documented and expected behavior is to use the `Init` operator unless overriden by explicitly supplying an argument via `--init`. Is this correct, then I don't see how printing out that the expected behavior is happening can help users who encounter unexpected behavior :) But if my assumption is wrong, and this behavior is not documented and expected, then I'd be in favor of changing the behavior or adjusting the expectations instead of adding to the (already quite verbose) output?

perhaps there are some dimensions of the info logged here that I'm not appreciating tho.

In any case, I wouldn't block on this :)

Reply:
it's just that not everyone might read the docs before trying the tool... at least I do this quite often;) In general, I don't have any strong opinion here; whatever @Igor decides is fine with me. 

What would be nice is at some point to discuss between all of us the general guidelines on what we log, how much, and in what way.

Reply:
Let's make it `debug` for now. The output is sometimes too detailed.